app/views/users/show.html.erb

```diff
@@ -1 +1,19 @@
-<h2>Show page for all users</h2>
+<div class="user-info flex">
+  <%= render 'shared/user', user: @user %>
+
+  <div class='bio'>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <div class='card'>
+      <% @user.three_recent_posts.each do |post| %>
+        <a href="/users/<%= @user.id %>/posts/<%= post.id %> ">
+          <%= render 'shared/post', post: post %>
```

Comment:
- [ ] Kindly style the post part for the user show page to match the wireframe: [https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABY4RVENCWGONH3SFHF5DZB4Y4J3SSA](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABY4RVENCWGONH3SFHF5DZB4Y4J3SSA)

  ![image](https://user-images.githubusercontent.com/54848363/205269146-df0bd50e-78e6-4da6-bc23-6d09f0911310.png)